client/src/controllers/ActionController.ts

```diff
@@ -0,0 +1,53 @@
+import { Controller } from '@hotwired/stimulus';
+import { WAGTAIL_CONFIG } from '../config/wagtailConfig';
+
+/**
+ * <button type="submit" class="button no"
+ * data-controller="w-action"
+ * data-action="click->w-action#post"
+ * data-w-action-redirect-value="true"
+ * data-w-action-redirect-url-value="{% url 'wagtailadmin_home' %}"
```

Comment:
Yup, that’s exactly what we’re after. Only thing I’m not sure of is whether we need the two data- attributes or just the one that does both things.

Reply:
Just the data-w-action-redirect-value attribute does the work already. there is no need for the second redirect attribute